### DISCUSSION TITLE
How to delete previous line(s)?

### DISCUSSION THREAD

**Discussion by JayJoeSeventh** [highfestiva/finplot#358] [Q&A] [answered]
I used fplt.add_line to draw stoploss line.
However, as time goes by, I update the plot with new stoploss line, but the old line (old stoploss) is still there.
How to delete previous lines?

**Comment by highfestiva** [comment, marked as answer]
This fix is not released yet, so please patch from github to get it working: https://github.com/highfestiva/finplot/commit/6e36fe01ddcd496a83f8b53307d2e25adf71e554. Then you can do like so:

```
finplot.remove_primitive(your_line)
```
